packages/core/components/RichTextEditor/Render.tsx

```diff
@@ -0,0 +1,64 @@
+import {
+  Extensions,
+  generateHTML,
+  generateJSON,
+  JSONContent,
+} from "@tiptap/react";
+import { useMemo } from "react";
+
+import getClassNameFactory from "../../lib/get-class-name-factory";
+import { defaultExtensions } from "./extensions";
+import styles from "./styles.module.css";
+
+const getClassName = getClassNameFactory("RichTextEditor", styles);
+
+export function Render({
+  content,
+  extensions = [],
+}: {
+  content: string | JSONContent;
+  extensions?: Extensions;
+}) {
+  const loadedExtensions = useMemo<Extensions>(
+    () => [...defaultExtensions, ...extensions],
+    [extensions]
+  );
```

Comment:
Hm, so does the user have to provide their tiptap extensions into the rendered user environment?

It's possible they might want to strip out unnecessary internal packages.